producer/proto/render.go

```diff
@@ -61,11 +61,152 @@ var (
 		0x86dd: "IPv6",
 	}
 	protoName = map[uint32]string{
+		0:   "HOPOPT",
 		1:   "ICMP",
+		2:   "IGMP",
+		3:   "GGP",
+		4:   "IPv4",
+		5:   "ST",
 		6:   "TCP",
+		7:   "CBT",
+		8:   "EGP",
+		9:   "IGP",
+		10:  "BBN-RCC-MON",
+		11:  "NVP-II",
+		12:  "PUP",
+		13:  "ARGUS",
+		14:  "EMCON",
+		15:  "XNET",
+		16:  "CHAOS",
 		17:  "UDP",
-		58:  "ICMPv6",
+		18:  "MUX",
+		19:  "DCN-MEAS",
+		20:  "HMP",
+		21:  "PRM",
+		22:  "XNS-IDP",
+		23:  "TRUNK-1",
+		24:  "TRUNK-2",
+		25:  "LEAF-1",
+		26:  "LEAF-2",
+		27:  "RDP",
+		28:  "IRTP",
+		29:  "ISO-TP4",
+		30:  "NETBLT",
+		31:  "MFE-NSP",
+		32:  "MERIT-INP",
+		33:  "DCCP",
+		34:  "3PC",
+		35:  "IDPR",
+		36:  "XTP",
+		37:  "DDP",
+		38:  "IDPR-CMTP",
+		39:  "TP++",
+		40:  "IL",
+		41:  "IPv6",
+		42:  "SDRP",
+		43:  "IPv6-Route",
```

Comment:
IPv6-EH-Routing maybe?

Reply:
`IPv6-Route` is the official IANA name, same as with `ICMPv6` vs. `IPv6-ICMP` - we can deviate from the standard, as you like.  
But in my opinion, we should stick to the standard.

Reply:
+1 to sticking to official IANA protocol names

Reply:
Good point